scripts/mysql_system_tables.sql

```
@@ -26,6 +26,7 @@
 
 set @have_innodb= (select count(engine) from information_schema.engines where engine='INNODB' and support != 'NO');
 set @is_mysql_encrypted = (select ENCRYPTION from information_schema.INNODB_TABLESPACES where NAME='mysql');
+set @ddse= (select @@default_dd_storage_engine);
```

ddse == ROCKSDB && is_mysql_encrypted == true combination should be diagnosed and rejected?

---

would you mind elaborating this? I didn't quite follow.

---

This same script allows creating an instance with mysql tablespace in InnoDB being encrypted, which MyRocks would not support. I believe we should return an error if the DD engine is MyRocks and mysql tablespace encryption is requested.